omod/src/main/java/org/openmrs/module/patientdashboard/web/controller/OPDEntryController.java

```diff
@@ -247,7 +247,7 @@ public String firstView(
 			model.addAttribute("admittedStatus", "Admitted");
 		}
 		
-		IpdPatientAdmission ipdPatientAdmission=ipds.getIpdPatientAdmissionByPatient(patient);
+		IpdPatientAdmission ipdPatientAdmission=ipds.getIpdPatientAdmissionByPatientId(patient);

```

Comment:
What informed this change? is there any benefits of getting a patient by Id as oppossed to patient obeject?

Reply:
> What informed this change? is there any benefits of getting a patient by Id as oppossed to patient obeject?

I added 'Id' to it because that is how it is referenced in hospitalcore module otherwise it was throwing an error though I can confirm again

 can not find symbol:   method getIpdPatientAdmissionByPatient(org.openmrs.Patient)
[ERROR]   location: variable ipds of type org.openmrs.module.hospitalcore.IpdService